src/cephadm/cephadm

```diff
@@ -2024,7 +2024,8 @@ def command_bootstrap():
                     fsid,
                     args.output_config,
                     args.output_keyring))
-
+    logger.info('Please consider enabling telemetry to help improve Ceph:\n\n'
+                '\thttps://docs.ceph.com/docs/master/mgr/telemetry/\n')
```

Comment:
Instead of just pointing to the docs, we should also provide a **real** command to activate telemetry.